subsys/bluetooth/mesh/access.c

```diff
@@ -2122,6 +2150,24 @@ int bt_mesh_comp_store(void)
 			LOG_DBG("Stored composition data P1");
 		}
 	}
+
+	if (IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_2)) {
+		net_buf_simple_reset(&buf);
+		err = bt_mesh_comp_data_get_page_2(&buf);
+
+		if (err) {
+			LOG_ERR("Failed to read composition data P2: %d", err);
```

Comment:
When you actually create PR, it may be worth to combine all these error messages. There is quite a bit of repeated code now. If you want to keep the structure, may be rewording these strings will optimize flash a bit. E.g. "read err cdp2: %d".